docs/concepts/cluster_queue.md

```diff
@@ -132,9 +196,9 @@ steps:
    didn't specify them already.
 
    For example, for a [batch/v1.Job](https://kubernetes.io/docs/concepts/workloads/controllers/job/),
-   Kueue adds the labels to `.spec.template.spec.nodeSelector`. This guarantees
-   that the workload Pods run on the nodes associated to the flavor that Kueue
-   decided that the workload should use.
+   Kueue adds the labels to the `.spec.template.spec.nodeSelector` field. This
+   guarantees that the workload Pods run on the nodes associated to the flavor
+   that Kueue decided that the workload should use.
```

Comment:
```suggestion
   Kueue adds the labels to the `.spec.template.spec.nodeSelector` field. This
   guarantees that the workload Pods can only be scheduled on the nodes targeted by the flavor
   that Kueue assigned to the workload.
```

Reply:
nice, I was having a hard time with this one :)